src/components/Fetch.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import './Fetch.css';
+
+export default function Fetch({ value, onSearchItem }) {
+  const [listItem, setListItem] = useState([]);
```

Comment:
This should be in plural since it is an array of objects. It is easier to read and everyone directly knows, that it's more than just one object.
```suggestion
  const [listItems, setListItems] = useState([]);
```